Samples/Samples/ViewModel/CalendarViewModel.cs

```diff
@@ -177,7 +178,7 @@ async void RefreshEventList(string calendarId = null, DateTime? startDate = null
         {
             startDate = StartDatePickersEnabled && !startDate.HasValue ? (DateTime?)StartDate.Date + StartTime : startDate;
             endDate = (EndDatePickersEnabled && !endDate.HasValue) ? (DateTime?)EndDate.Date + EndTime : endDate;
-            if (CalendarList.Count == 0)
+            if (!CalendarList.Any())
```

Comment:
ok, so an interesting debate to be had on this change:
- Any() is more generic and if the underlying type of CalendarList changes, Any will always work.
- Count is more efficient in most cases because most list structures will have a count of items anyhow, so it just returns that value
- To be completely safe, you should really do CalendarList?.Any() but then you need to force it to be a bool instead of bool? so you make it (CalendarList?.Any()??false) which seems much more long winded than just .Count==0. 
- To avoid the need to null check CalendarList, you could make it readonly.
@mkieres any thoughts on what we'd prefer to see here?

Reply:
lol - yep, agreed, let's make the list readonly and go with Any

Reply:
Maybe I misunderstood what you meant by make the list readonly, since I need to be able to change the calendar list based on the calendars you have, how I could I go about making it readonly?

Reply:
just repopulate the collection instead of creating a new instance

Reply:
@ScottBTR are you going to make the change to the collection in this PR, or in another?

Reply:
will apply it to this PR just not sure I understand how you repopulate a readonly property, probably something simple, is there a simple example online of what you mean by this?

Reply:
I don't think we're thinking of anything fancy: call Clear on the collection and then use Add (or ideally AddRange if it exists) to repopulate the collection

Reply:
After discussion with Mikolaj I understand where the confusion was, I was unaware that once you made a list readonly you still had the ability to .Add or .Clear, I thought once a property became readonly you were only able to populate it on instantiation via constructor/property set to a static field.